src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -57,6 +57,7 @@ enum MessageCase {
 }
 
 interface IProps {
+    roomId: string;
```

Comment:
Please can you give this some documentation - and possibly add some for `room` as well. Why are both properties needed?

Reply:
okay, it is updated